README.md

```diff
@@ -226,9 +226,22 @@ In your application, you can use the following configuration:
 
 <summary>Read the cookbook</summary>
 
-In order to use RabbitMQ and its dashboard, you should add the following content
-to the `docker-compose.yml` file:
+In order to use RabbitMQ and its dashboard, you should add a new container:
+
+```Dockerfile
+# services/rabbitmq/Dockerfile
+FROM rabbitmq:3-management-alpine
+
+COPY rabbitmq.conf /etc/rabbitmq/
```

Comment:
Can you update the doc to follow this convention: https://github.com/jolicode/docker-starter/blob/main/infrastructure/docker/services/router/Dockerfile#L3

I think it's easier to read, and I usually found the file I need to edit faster.

Reply:
I hesitated to use the convention but changed my mind after, my bad :stuck_out_tongue: I'll fix it :slightly_smiling_face: